tests/test_sns/test_sms_boto3.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import unicode_literals
```

Comment:
I think this filename has a typo: 'sms' -> 'sns'

Reply:
half and half, originally I thought the functions left were mostly sms methods but there more subscriber general ones so shall rename.

Reply:
Actually, looking at it I can distribute these tests to other files which would suit them better.

Reply:
Ha! I didn't realize it. I guess I just got confused. I trust your naming intuition here.

Reply:
No worries, too late ha, I distributed them across the other test files as they too fit under those categories :-)